src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, useState } from 'react';
+import { LoginProps } from './LoginForm.types';
+import * as Styled from './LoginForm.styles';
+import InputAdornment from '@mui/material/InputAdornment';
+import * as icons from '@mui/icons-material';
+
+const LoginForm: FC<LoginProps> = ({ onLogin }) => {
+  const [emailValue, setEmail] = useState('');
+  const [passwordValue, setPassword] = useState('');
+
+  const [emailError, setEmailError] = useState(false);
+  const [passwordError, setPasswordError] = useState(false);
+
+  const handleButtonClick = () => {
+    let good: boolean = true;
```

Comment:
Same here, boolean is not needed.

Reply:
Addressed in commit: [03298211be1510af94c17aaecbe74406efdebccf](https://github.com/Luvkitri/zpab-frontend/pull/2/commits/03298211be1510af94c17aaecbe74406efdebccf)